functions/webhooks/taskrouterCallback.protected.ts

```diff
@@ -107,6 +107,14 @@ export const handler = async (
           }),
         ),
       );
+
+      // This helper attaches the taskSid to channel attributes. This is used by endChat serverless function used by webchat
+      const addTaskHandlerPath =
+        Runtime.getFunctions()['helpers/addTaskSidToChannelAttributes'].path;
+      const addTaskSidToChannelAttributes = require(addTaskHandlerPath)
+        .addTaskSidToChannelAttributes as AddTaskSidToChannelAttributes;
+      await addTaskSidToChannelAttributes(context, event);
```

Comment:
The `resolve` function should moved _after_ this, to avoid race conditions (your chunk may not be called cause the [serverless callback](https://www.twilio.com/docs/serverless/functions-assets/functions/invocation#callback-function) exiting before).